src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -84,6 +84,99 @@ def to_tuple(self) -> Tuple[Any]:
         )
 
 
+@dataclass
+class Blip2ImageTextMatchingModelOutput(ModelOutput):
```

Comment:
Not sure if feasible, but it'd be nice to match the output class of CLIP, which is also an image-text matching model. It consists of the following keys:
* loss
* logits_per_image (this I assume is the itm_score)
* logits_per_text (this I assume is the itm_score transposed)
* and some other keys which are CLIP-specific.

Making sure that Blip2ForImageTextRetrieval matches this would allow it to be added to the zero-shot image classification pipeline, which relies on this output key: https://github.com/huggingface/transformers/blob/bbaa8ceff696c479aecdb4575b2deb1349efd3aa/src/transformers/pipelines/zero_shot_image_classification.py#L143

Reply:
Otherwise we will have a hard time adding BLIP-2 support to the zero-shot image classification pipeline.

Reply:
Hi, i updated the output to match CLIP output

Reply:
Hi @NielsRogge, i updated the output to match CLIP output, but this PR is not being updated with my latest commits